en/faq/adback.rst

```diff
@@ -0,0 +1,111 @@
+FAQ
+===
+
+What is AdBack?
+---------------
+AdBack is a professional solution that analyzes your Internet audience using adblockers. It collects, decrypts and alerts you about the evolution of this audience. It allows you to react, to inform and monetize this specific audience. 
+
+What is an adblocker user?
+--------------------------
+Today, there are different ways to block ads. An adblocker user is a visitor using one of the solutions below:
+a browser extension: several extensions exist such as Adblock Plus, AdBlock, Ghostery, uBlock, uBlock Origin or Crystal on mobile iOS
+an antivirus software
+a personal firewall via the antivirus
+a professional firewall via the company 
+
+Why should I install AdBack?
+----------------------------
+By installing AdBack, you’ll have access to a highly technological tool which is not detected by adblockers, to real-time informations, to a maximized and constantly evolving script and also to a wide range of solutions to dialogue with the adblocker user audience.
+
+What benefits can I expect by using AdBack?
+-------------------------------------------
+With AdBack, understand your adblocker users and re-establish the dialogue between you and them. With AdBack, win back 28 to 60% of your advertising revenue lost because of adblockers.
+
+Who is AdBack for?
+------------------
+AdBack is for every content publisher who wants to understand and monetize its adblocked audience: media websites, regional magazines, bloggers. AdBack is also for e-commerce websites, advertisers, agencies who want to take an audit on their website, verify in real-time that adblocker users don’t impact the conversion rate and utilize tools to generate leads specifically on this audience.
```

Comment:
Could you set a hard limit on your lines.

This will allow us to have more precise comments (less word per lines) and will also improve the readability in our ide.